web/router.go

```diff
@@ -188,6 +188,7 @@ func v2Routes(app services.Application, engine *gin.Engine) {
 
 		txs := TxAttemptsController{app}
 		authv2.GET("/txattempts", txs.Index)
+		authv2.GET("/txattempts/:TxHash", txs.Show)
```

Comment:
The show path for a tx attempt with an id would be `/txattempts/:TxAttemptID` which conflicts with this route.

Would this be better server as
1. New show path route for transactions at `/transactions/:TxHash`
2. Filter by parameter `/txattempts?hash=:TxHash`


(1) Implies a new controller `TransactionsController#Show` which is what I'm personally leaning towards. For all intents and purposes, you're returning a `Tx` not a `TxAttempt`

Reply:
+1 to a `TransactionsController`

I just realized the `/v2/txattempts#index` endpoint doesn't have `from` or `to` fields so we also need either:

- A `/v2/transactions#index` endpoint 
- Return the transaction as an association on the `/v2/txattempts#index` endpoint

Reply:
Just to clarify @rupurt, are you saying

* A /v2/transactions#index endpoint [with filters param for `from` or `to`?]
* Return the transaction as an association on the /v2/txattempts#index endpoint [so that we can do the filtering client side in redux?]

Reply:
@dimroc I believe that @rupurt is referring to [the story he's working](https://www.pivotaltracker.com/story/show/163042684) on specifying a `from` and `to` column.

I'd been thinking we should eventually repurpose the TxAttempts resource into a Transactions. TxAttempts is unique to Chainlink, and probably doesn't fit the mental model for most people of how a blockchain works. I was thinking of reframing attempts as Unconfirmed Transactions, because that's a bit more intuitive. At least one downside to that would be that it breaks our ubiquitous naming.

@rupurt had suggested that we nest attempts in transactions which does fit our model. I like that idea, it matches our modeling. We can hide attempts when there is only one, which is likely 99% of the time, or once the transaction is considered safe.

Reply:
@dimroc multiple addresses are not taken into account. @thoughtmerchant do you have thoughts on how you want to display that?

Reply:
Moved transaction show over to it's own controller, will move the transaction attempts index over in a subsequent story.